src/test/SmartComponents/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -213,3 +281,41 @@ describe('Step Review', () => {
         verifyCancelButton(cancel);
     });
 });
+
+describe('Click through all steps', () => {
+    beforeEach(() => {
+        renderWithReduxRouter(<CreateImageWizard />);
+    });
+
+    test('with valid values', async () => {
+        const next = screen.getByRole('button', { name: /Next/ });
+
+        // select release
+        userEvent.selectOptions(screen.getByTestId('release-select'), [ 'rhel-8' ]);
+        next.click();
+
+        // select upload target
+        await screen.findByTestId('upload-destination');
+        userEvent.selectOptions(screen.getByTestId('upload-destination'), [ 'aws' ]);
```

Comment:
@Gundersanne this yields `TypeError: Cannot read property 'access_key_id' of null`  and I'm not sure if I'm messing up something or it is a legitimate error, see:
https://travis-ci.com/github/osbuild/image-builder-frontend/builds/200385002. Please advise.

Reply:
hmm, that might be a legitimate error. The wizard should pass state.upload in it's entirety to the uploadcomponent. Which means `setUpload` somehow gets called where it updates the state with an object where `.options` doesn't exist

Reply:
:beetle: :beetle: :beetle: yay

Reply:
@Gundersanne IDK if I fixed this issue with my second commit or not but I am seeing a different error now:
`ReferenceError: insights is not defined`, see https://travis-ci.com/github/osbuild/image-builder-frontend/builds/201236611